caffe2/mobile/contrib/arm-compute/operators/elementwise_sum_op.cc

```diff
@@ -27,6 +27,9 @@ bool GLSumOp<T>::RunOnDevice() {
   if (first_run_) {
     A_ = GLContext::getGLTensor<T>(Ablob);
     B_ = GLContext::getGLTensor<T>(Bblob);
+  } else {
```

Comment:
you can eliminate the `if` right? on first run `A_.release()` will return a nullptr which is what you'd want